File: vue/ceseat/src/types/Order.ts
import EOrderState from '@/types/EOrderState';
export default interface OrderPricing {
    paid?: boolean;
    payment_card_id?: number;
    comission_fees?: number;
    restaurant_fees?: number;
    delivering_fees?: number;
    discount?: number;
    total?: number;
}

export default interface OrderMenu {
    id?: any;
    name?: string;
    description?: string
    price?: number
    products?: Array<any>

}

export default interface Order {
    id?: any;
    date?: any;
    restaurant_id?: any;
    customer_id?: any;
    deliverer_id?: any;
    status?: string;
    pricing?: OrderPricing;
    menus?: Array<OrderMenu>;
}

File: CeseatConnect/CeseatConnect/CeseatConnect/Order.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace CeseatConnect
{
    class Order
    {
        public string id { get; set; }
        public string date { get; set; }
        public string restaurant_id { get; set; }
        public string customer_id { get; set; }
        public string deliverer_id { get; set; }
        public string status { get; set; }
        public Pricing pricing { get; set; }
        public bool delivered { get; set; }
        public Menus[] menus { get; set; }
    }

    class Pricing
    {
        public bool paid { get; set; }
        public string payment_card_id { get; set; }
        public float comission_fees { get; set; }
        public float restaurant_fees { get; set; }
        public float delivering_fees { get; set; }
        public float discount { get; set; }
        public float total { get; set; }
    }

    class Menus
    {
        public Products[] products { get; set; }
        public string name { get; set; }
        public string description { get; set; }
        public string restaurant_id { get; set; }
        public float price { get; set; }
        public bool available { get; set; }
        public string id { get; set; }
    }

    class Products
    {
        public string id { get; set; }
        public string name { get; set; }
        public string description { get; set; }
        public string restaurant_id { get; set; }
        public bool available { get; set; }
        public float price { get; set; }
        public string[] ingredients { get; set; }
    }
}
